packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -851,6 +859,11 @@ const RichTextContainer = compose( [
 				selectionStart.clientId === clientId &&
 				selectionStart.attributeKey === identifier
 			);
+		} else {
+			isSelected = isSelected && (
```

Comment:
This change (the addition of the "else" block with its setting of the `isSelected` flag) seems to be causing https://github.com/wordpress-mobile/gutenberg-mobile/issues/1031. @SergioEstevao , do you recall what this change was added to do? I'm going to open a PR to remove it but, I'm afraid I might be missing some context and the removal might add some other problem.

Reply:
@SergioEstevao added a comment about this here: https://github.com/WordPress/gutenberg/pull/15833#issuecomment-496262819